src/Ocelot/Configuration/Creator/AggregatesCreator.cs

```diff
@@ -26,6 +25,12 @@ public List<Route> Create(FileConfiguration fileConfiguration, List<Route> route
 
         private Route SetUpAggregateRoute(IEnumerable<Route> routes, FileAggregateRoute aggregateRoute, FileGlobalConfiguration globalConfiguration)
         {
+            if (!aggregateRoute.UpstreamHttpMethod.Any())
+            {
+                // Default method to Get for standard use case
+                aggregateRoute.UpstreamHttpMethod.Add(HttpMethod.Get.ToString());
+            }
+

```

Comment:
We have to unit test this code.

Reply:
And what's the goal of having static field, Raynald?
I have another alternative which is static too &rarr; `HttpMethods.Get` in the `Microsoft.AspNetCore.Http` namespace 😉
This is fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1389/commits/02a3d991c4eb4d8ae04f8ff364046bd7a35aceb3 ✔️

Reply:
@RaynaldM, I've implemented your concept of a single global setting. The commit is https://github.com/ThreeMammals/Ocelot/pull/1389/commits/98083a1e3cb503723e5db8e8ea0c8b564dced01b. However, this setting can only be overridden in the C# code; there's no corresponding JSON setting in `ocelot.json`.
Also, I chose to add the global static property to the `FileAggregateRoute` class rather than the `AggregatesCreator` because it appears to be the most appropriate place for its definition.

Reply:
Unit tests commit is https://github.com/ThreeMammals/Ocelot/pull/1389/commits/7e071939f553d0054179740f4d4b2929e64942ca ✔️